### PR TITLE
Add priority to the status tooltip of the Project List

### PR DESCRIPTION
**Pull request by JIY0UNG**
## Description
The project list shows priority through the background color of Status column.
Red means Priority1, and yellow means Priority2.
However, it was impossible to know the meaning of these colors.
So I added the information of each color in the tooltip.
(Related issue: https://github.com/fosslight/fosslight/issues/581)

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/admin/project/list-js.jsp [hunk 1]
```diff
@@ -38,7 +38,9 @@
 	                   +"<dl><dt><span class=\"iconSt request\">Request</span>Request</dt></dl><br>"
 	                   +"<dl><dt><span class=\"iconSt review\">Review</span>Review</dt></dl><br>"
 	                   +"<dl><dt><span class=\"iconSt complete\">Complete</span>Complete</dt></dl><br>"
-	                   +"<dl><dt><span class=\"iconSt drop\">Drop</span>Drop</dt></dl><br>"
+	                   +"<dl><dt><span class=\"iconSt drop\">Drop</span>Drop</dt></dl><br>" 
+                       +"<dl><dt><span class=\"priority priority_1\"></span>Priority1</dt></dl><br>"
+                       +"<dl><dt><span class=\"priority priority_2\"></span>Priority2</dt></dl><br>"
 	                   +"</div>",
```
Comment (soimkim) [inline src/main/webapp/WEB-INF/views/admin/project/list-js.jsp:44]: Please check the indentation.

Comment (JIY0UNG) [reply src/main/webapp/WEB-INF/views/admin/project/list-js.jsp]: Modified it!

File: src/main/webapp/WEB-INF/views/admin/project/list-js.jsp [hunk 2]
```diff
@@ -961,7 +963,7 @@
 			var chk = $("#list").jqGrid("getGridParam", "selarrrow").length;
 
 			if(chk > 0){
-				$(".selectSet").find("strong").text($("#changeDivisionPop select[name=division] option:first").text());
+				$("#changeDivisionSelect").find("strong").text($("#changeDivisionPop select[name=division] option:first").text());
```
Comment (soimkim) [inline src/main/webapp/WEB-INF/views/admin/project/list-js.jsp:966]: Please remove irrelevant edits.

File: src/main/webapp/WEB-INF/views/admin/project/list-js.jsp [hunk 3]
```diff
@@ -39,7 +39,9 @@
 	                   +"<dl><dt><span class=\"iconSt review\">Review</span>Review</dt></dl><br>"
 	                   +"<dl><dt><span class=\"iconSt complete\">Complete</span>Complete</dt></dl><br>"
 	                   +"<dl><dt><span class=\"iconSt drop\">Drop</span>Drop</dt></dl><br>"
-	                   +"</div>",
+					   +"<dl><dt><span class=\"priority priority_1\"></span>Priority1</dt></dl><br>"
+					   +"<dl><dt><span class=\"priority priority_2\"></span>Priority2</dt></dl><br>"
+					   +"</div>",
```
Comment (soimkim) [inline src/main/webapp/WEB-INF/views/admin/project/list-js.jsp:44]: Please check indentation.

**Pull request merged**